src/main/java/com/exadel/discount/service/impl/TagServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.exadel.discount.service.impl;
+
+import com.exadel.discount.dto.TagDTO;
+import com.exadel.discount.entity.Tag;
+import com.exadel.discount.exception.NotFoundException;
+import com.exadel.discount.mapper.TagMapper;
+import com.exadel.discount.repository.TagRepository;
+import com.exadel.discount.service.TagService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class TagServiceImpl implements TagService {
+
+    private final TagRepository tagRepository;
+    private final TagMapper tagMapper;
+
+    @Override
+    public List<TagDTO> getAllTags() {
+        log.info("Getting list of all Tags");
+
+        List<Tag> allTags = tagRepository.findAll();
+        log.info("Successfully got list of all Tags");
```

Comment:
I believe it should be debug level. And other similar messages the same

Reply:
+